imap/jmap_sieve.c

```diff
@@ -2257,7 +2259,6 @@ static int jmap_sieve_test(struct jmap_req *req)
         }
     }
 
-    if (interp_ctx.cstate) conversations_commit(&interp_ctx.cstate);
```

Comment:
The convdb is opened for us by jmap_api. If we commit it here, we'll crash later because jmap_api.c will try to commit it too